app/controllers/api/channels_controller.rb

```diff
@@ -4,34 +4,29 @@ def index
   end
 
   def create
-    current_user_id = current_user.id
-
-    permissions = [current_user_id]
+    permissions = [current_user.id]
```

Comment:
poking around in the channels_controller i went to Channel model, to understand some stuff and saw that you are calling #sort method in a couple places, kind of like Javascript invocation though, `.sort()`. Why?

Reply:
The jS invocation is just because I was in a jS mindset when I wrote it. The basic point of this was to see if a dm with the given set of users already exists. As for why sort them at all, I figured that was the simplest way to compare the two arrays, but I guess if all of the ids are going to be unique I could just use a a & b = a comparison.